i18n/en/docusaurus-plugin-content-docs/current/guides/tech/with-nextjs.mdx

```diff
@@ -1,72 +1,73 @@
 ---
 sidebar_position: 10
 ---
-# Usage with NextJS
+# Usage with Next.js
 
-It is possible to implement FSD in a NextJS project, but conflicts arise due to differences between the requirements for the NextJS project structure and the principles of FSD in two points: 
+It's possible to implement FSD in a Next.js project, but conflicts arise due to differences between Next.js project structure requirements and FSD principles.
 
-- Routing files in the `pages` layer
-- Conflict or absence of the `app` layer in NextJS
+## Conflict between FSD and Next.js in the `app` layer
 
-## Conflict between FSD and NextJS on `pages` layer {#pages-conflict}
+Next.js suggests using the `app` folder to define application routes. Next.js expects files in the `app` folder to correspond to URLs.
+This routing mechanism **does not align** with the FSD concept, as it's not possible to maintain a flat slice structure with such a routing mechanism.
 
+The approach is to move the Next.js `app` folder to the project root and import FSD pages into the Next.js `app` folder.
+This preserves the FSD project structure inside the `src` folder.
 
-NextJS suggests using the `pages` folder to define application routes. NextJS expects files in the `pages` folder to match URLs.
-This routing mechanism **does not correspond** to the FSD concept, since it is not possible to maintain a flat slice structure in such a routing mechanism.
+You should also add a `pages` folder to the project root because App Router is compatible with Pages Router.
+The best way is to put an empty `.gitkeep` or `README.md` file inside describing the need to save this folder.
```

Comment:
suggestion (non-blocking): I would focus the recommendation to the README file instead of `.gitkeep` as it's not obvious why this empty folder is needed, so project docs could really come in handy later down the line